admin_manual/configuration/files/external_storage/auth_mechanisms.rst

```diff
@@ -39,6 +39,21 @@ does not work.
    (See :doc:`../../server/background_jobs_configuration`.) Be
    aware that the Ajax cron mode is triggered by browsing the ownCloud Web GUI.
 
+Known Limitations
+~~~~~~~~~~~~~~~~~
+
+Please be aware that because of the way that credentials are stored in ownCloud, any operations must be performed by the logged-in mount owner, because credentials are not stored anywhere.
+As a result, there are three known limitations.
+These are:
+
+#. Directly sharing the storage or any its sub-folders will fail. They will not appear to the recipient, because the mount cannot be set up due to missing credentials.
+#. Any background task operating on the storage, such as background scanning.
+#. Any :doc:`occ command <../configuration/server/occ_command>` that operates on the storage like occ files:scan will fail.
```

Comment:
does fail mean that there will be a error message or does it just not do it without notifying the user.

Reply:
@PVince81, @phil-davis can you clarify?

Reply:
I don't know and don't have a quick setup to try this out right now. How about changing to "will have no effect" which could mean any of "fail", "show an error" or "do nothing"

Reply:
I have not tried this directly myself, so not confident to advise!